syntax = "proto3";

service ChatService {
    rpc ConsumeClientMessage(ClientMessage) returns (BotReply);
}

message ClientMessage {
    string text = 1;
}

message BotReply {
    string text = 1;
    string analysis = 2;
}